### ISSUE TITLE
This file (....) is already opened in the editor but with a different casing. IntelliSense features will be disabled on this copy of the file.

### ISSUE THREAD

**Issue by XuChen-Zhao** [microsoft/vscode-cpptools#3818]
I used vscode as a editor.When I updated the C/C++ to the 0.24.0 , there will be some warning "This file (....) is already opened in the editor but with a different casing. IntelliSense features will be disabled on this copy of the file. ". But Iopend this file only once.I could find out how to disable the warning and how to fix this warning .
When I install the 0.23.1 version, there is not this warning.
Thank you very much.

**Comment by CMinHong** [comment]
啊我也遇到同样的问题，现象是鼠标停留到某个方法上就这样了。

**Comment by XuChen-Zhao** [comment]
是的，目前还没有找到解决方法。我已经将C/C++插件改为0.23.1版本，并取消了插件的自动更新。






------------------ 原始邮件 ------------------
发件人: "CMinHong"<notifications@github.com>;
发送时间: 2019年6月24日(星期一) 上午10:35
收件人: "microsoft/vscode-cpptools"<vscode-cpptools@noreply.github.com>;
抄送: "赵旭辰"<348961724@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [microsoft/vscode-cpptools] This file (....) is already opened inthe editor but with a different casing. IntelliSense features will be disabledon this copy of the file. (#3818)




啊我也遇到同样的问题，现象是鼠标停留到某个方法上就这样了。

**Comment by michelleangela** [comment]
Thank you for reporting the issue.

This is a duplicate of #3786
